File: Python/IsomorphicStrings.py
def isIsomorphic(s,t):
    metaMap = {}

    for i in range(len(s)):
        if s[i] in metaMap:
            if metaMap[s[i]] != t[i]:
                return False
        else:
            if t[i] in metaMap.values():
                return False
            metaMap[s[i]] = t[i]

        return True

File: Javascript/IsomorphicStrings.js
var isIsomorphic = function(s,t){
    let metaMap = new Map()

    for(let i =0;i<s.length;i++){
        if(metaMap.has(s[i])){
            if(metaMap.get(s[i]) != t[i]){
                return false
            }
        }
        else{
            for(let [key,value] of metaMap){
                if(value === t[i]){
                    return false
                }
            }
            metaMap.set(s[i],t[i])
        }
    }
    return true
}